tox.ini

```diff
@@ -78,7 +78,7 @@ commands =
     bash -c 'black --check --config pyproject.toml {env:LINT_FILES_TO_CHECK}'
     python .circleci/modernize/modernize.py -j 2
     bash -c 'flake8 --config lint-configs/python/.flake8 {env:LINT_FILES_TO_CHECK}'
-    bash -c 'pylint -E --rcfile=./lint-configs/python/.pylintrc --load-plugins pylint_plugins.bundled_imports_checker {env:LINT_FILES_TO_CHECK}'
```

Comment:
Yeah, it's indeed much better we centralize the settings inside the shared lint configs (where possible) :+1: